synapse_data_warehouse/synapse/dynamic_tables/V2.40.0__acl_latest_refactored.sql

```diff
@@ -0,0 +1,43 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+
+CREATE OR REPLACE DYNAMIC TABLE ACL_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS 
+    WITH dedup_acl AS (
+        SELECT
+            *,
+            parse_json(resource_access) as acl,
```

Comment:
Interpreting the fields in `resource_access` isn't necessary to determine the latest snapshot, so we're making Snowflake call `parse_json` unnecessarily over a pretty significant number of snapshots which we're just going to drop anyways.

Can we move `parse_json` into the `LATERAL FLATTEN` function instead?

Reply:
Resolved!